thesis-de/content-de.tex

```diff
@@ -2942,6 +2943,34 @@ \section{Unit-Tests}
 \end{outline}
 \mr%
 
+\section{Acceptance Tests}
+Ein Akzeptanz Test prüft den zuvor erstellten Code in einer eigens dafür
+gestarteten virtuellen Maschine oder Container mit einem Betriebssystem auf die
+Funktionalität der einzelnen Module. Hierzu wurden schon bereits vorhandene
+Test Szenarien von \texttt{testkitchen} und \texttt{serverspec} verwendet und
+diese auf die im Projekt verwendeten Module angepasst. Der Code wird innerhalb
+dieser Tests ausgeführt und geprüft, ob die Funktionalität der einzelnen Module
+für den Benutzer gegeben ist. Diese Szenarien prüfen beispielsweise, ob die
+Erhaltung der Daten, sowie speichern dieser Daten in der Datenbank und
+Visualisierung der Graphen funktioniert. Es wird hier nicht geprüft, ob das
+System stabil läuft sondern lediglich, ob die einzelnen Funktionen vorhanden
+sind und funktionieren. Die Ausgabe erfolgt hier entweder in Bestanden oder
+nicht Bestanden. Anschließend kann der Entwickler diesen Code erneut anpassen
+und testen. Nachfolgend sind zwei Tests für einen Akzeptanztest für das Modul
+Collectd angehangen. Der vollständige Test ist dem Anhang beigefügt.
+
+\begin{listing}[H]
+  \inputminted{ruby}{../listings/acceptance-test.txt}
+  \caption{Zwei Akzeptanztests für collectd}
+\end{listing}
+
+Wie auch für die~\ref{sec:unit-tests} werden Akzeptanztests für verschiedene
+Betriebsysteme programmiert. Anders als bei~\ref{sec:unit-tests} werden diese
+nicht in einer \texttt{Metadata.json} hinterlegt, sondern jedes Betriebssystem
+erhalt eine eigenen Datei, in der definiert ist, mit welcher virtuellen
+Maschine oder Container dieser Code ausgeführt werden soll.
```

Comment:
was genau meinst du damit? die Datei in den Anhang? soll ich einen Beispiel hiervon in den Anhang tun?